<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Układ równań liniowych:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>69</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arytmetyka:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Wprowadź dane</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <item>
     <property name="text">
      <string>zmiennopozycyjna</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>przedziałowa (podana liczba)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>przedziałowa (podany przedział)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="solveButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Oblicz </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <item>
     <property name="text">
      <string>z macierzą symetryczną</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>z macierzą trójdiagonalną metodą Crouta</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
